sdk/src/wallet/account/mod.rs

```diff
@@ -191,7 +189,7 @@ impl Account {
 
     // Get the Client
     pub fn client(&self) -> &Client {
-        &self.client
+        &self.wallet.client
```

Comment:
should we call the client() here instead?

Reply:
That is the `client()` fn?